locals.tf

```diff
@@ -5,23 +5,32 @@ locals {
 
 locals {
   merged_private_link_private_dns_zones = merge(var.private_link_private_dns_zones, var.private_link_private_dns_zones_additional)
-  private_link_private_dns_zones_replaced_regionCode_map = { for k, v in local.private_link_private_dns_zones_replaced_regionName_map : k => {
```

Comment:
this was referring to `local.private_link_private_dns_zones_replaced_regionName_map` before it was defined which made the code harder to read so I reordered it to be defined first